packages/components/src/Calendar/CalendarRange/CalendarRange.module.scss

```diff
@@ -28,7 +28,7 @@ $cell-border-radius: 3px;
     }
   }
 
-  &:first-child {
+  &:nth-of-type(1) {
```

Comment:
rdpv9 has the nav inside the month container now so first-child didn't match

Reply:
Ah that's we're it got us 🤔